tests/integration/test_model.py

```diff
@@ -93,6 +93,25 @@
             assert app in model.applications
 
 
+@base.bootstrapped
+@pytest.mark.asyncio
+async def test_deploy_local_charm_operator_framework(event_loop):
+    from pathlib import Path
```

Comment:
If you remove this to the top of the file, we can drop the E402 changes to flake8?